apps/remix-ide/src/app/tabs/locales/en/udapp.json

```diff
@@ -186,5 +186,9 @@
   "udapp.EIP712-3": "In Remix, signing typed data is possible by right clicking (right click / Sign Typed Data) on a JSON file whose content is EIP-712 compatible.",
   "udapp.EIP712-create-template": "Create a JSON compliant with EIP-712",
   "udapp.EIP712-close": "Close",
-  "udapp.sign": "Sign"
+  "udapp.sign": "Sign",
+
+  "udapp.createDelegationAuthorization": "Create Delegation Authorization",
```

Comment:
Suggestion: It can be `Set code to EOA` as mentioned in EIP . cc: @ryestew 
Similarly other text can be updated